lib/system/mmdisp.nim

```diff
@@ -274,6 +275,9 @@ elif defined(gogc):
   proc realloc(p: pointer, newsize: Natural): pointer =
     doAssert false, "not implemented"
 
+  proc realloc0(p: pointer, oldsize, newsize: Natural): pointer =
+    doAssert false, "not implemented"
```

Comment:
It's better to not implement this via ``.error`` rather than fail at runtime.

Reply:
Done